bond-no-treasury/contracts/HectorBondV2NoTreasuryFTMDepository.sol

```diff
@@ -1109,14 +1166,16 @@ contract HectorBondV2NoTreasuryFTMDepository is OwnableUpgradeable {
         );
 
         uint256 fee = _amount.mul(feeBps).div(ONEinBPS);
-        tokenBalances[_principal][fundRecipient] += _amount.sub(fee);
+        tokenBalances[_principal][fundRecipient] = tokenBalances[_principal][
```

Comment:
@bejaxer isn't this the same as tokenBalances[_principal][fundRecipient] += _amount.sub(fee);?